src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -145,6 +145,20 @@ public void sortPersonsDesc() {
         addressBook.sortPersonsDesc(); // Call the sortPersonsDesc method from AddressBook
         updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS); // Refresh the filtered list after sorting
     }
+    /**
+     * Sorts the persons in the address book by putting the favourite tagged persons in front.
+     */
+    public void sortPersonsFavourite() {
+        addressBook.sortPersonsFavourite(); // Call the sortPersonsDesc method from AddressBook
+        updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS); // Refresh the filtered list after sorting
+    }
+    //    /**
```

Comment:
Remove commented code (lines 155-161)

Reply:
I might come back to use this, commented to not taken into account by codecov